lib/PuppeteerSharp/Target.cs

```diff
@@ -1,192 +1,191 @@
-using System;
-using System.Diagnostics;
-using System.Linq;
-using System.Threading.Tasks;
-using PuppeteerSharp.Helpers;
-
-namespace PuppeteerSharp
-{
-    /// <summary>
-    /// Target.
-    /// </summary>
-    [DebuggerDisplay("Target {Type} - {Url}")]
-    public class Target : ITarget
-    {
-        private readonly Func<bool, Task<CDPSession>> _sessionFactory;
-        private readonly bool _ignoreHTTPSErrors;
-        private readonly ViewPortOptions _defaultViewport;
-        private readonly TaskQueue _screenshotTaskQueue;
-        private readonly Func<TargetInfo, bool> _isPageTargetFunc;
-        private Task<Worker> _workerTask;
-
-        internal Target(
-            TargetInfo targetInfo,
+using System;

```

Comment:
new line change?

Reply:
Some files had a mix of CRLF and LF as newlines.
I've created an initial commit that aligns that for all files.